docs/Readme.md

```diff
@@ -1,140 +1,115 @@
-# Knora Documentation Generation #
+# Knora Documentation
 
-## 1 Install Python 3 ##
+## 1 Paradox (new) and Sphynx (old) Style Documentation
 
-On Mac OS X:
-
-```
-$ brew install python3
-```
-
-On Linux, use your distribution's package manager, or see [the Python web site](https://www.python.org).
+This folder contains the sources to the Knora documentation website published under
+[http://docs.knora.org](http://docs.knora.org). The `src` folder contains the following documentation sources:
 
-## 2 Installing Sphinx ##
+ - `src/jekyll`: The Knora Documentation Overview Website
+ - `src/paradox`: The Paradox-Mardown-based newer style Knora general documentation
+ - `src/sphinx`: The Sphinx-based older style Knora general documentation
+ - `src/api-v1`: The Knora JSON API V1 Request and Response Format documentation. Source can be found in `salsah1/src/typescript_interfaces`
+ - `src/api-v2`: The Knora JSON-LD API V2 Request and Response Format documentation.
+ - `src/api-admin`: The Knora JSON Admin API Request and Response format Swagger-based documentation.
 
-### 2.1 Globally ###
+We are currently in a transitioning phase, where we slowly move chapters from Sphinx to Paradox. The process of 
+moving chapter involves the following Pandoc command for converting `rst` documents to `md` documents:
```

Comment:
How good is the conversion? Could we convert everything all at once using a script? Or do the converted files require manual fixes?

Reply:
The links between documents is an extension of markdown through paradox. So those would need to be done manually. Also, we have some graphs embedded (via `dot`), which would need to be created manually and added to markdown manually. We could certainly try, and then see what happens.

Reply:
OK, let's merge this first.